test/test_web.rb

```diff
@@ -663,4 +663,46 @@ def app
       assert_equal 'nicehost.org', session_options[:host]
     end
   end
+
+  describe 'sidekiq web sessions options' do
+    include Rack::Test::Methods
+
+    def app
+      Sidekiq::Web.new
+    end
+
+    after { Sidekiq::Web.enable(:sessions) }
```

Comment:
@inkstak For tests that change the state of the application I recommend changing the `app` directly, as you can see it is an instance of `Sidekiq::Web`.

You can port these methods for usage on the instance and test them here. If the implementation on the instance level works and we assume that the instance implementation is a proxy of the class one (or vice-versa) then it's fine.